python/ray/serve/constants.py

```diff
@@ -74,3 +74,8 @@
 #: Number of times in a row that a replica must fail the health check before
 #: being marked unhealthy.
 REPLICA_HEALTH_CHECK_UNHEALTHY_THRESHOLD = 3
+
+# Key used to idenfity given json represents a serialized RayServeHandle
+SERVE_ASYNC_HANDLE_JSON_KEY = "__SerializedServeHandle__"
+# Key used to idenfity given json represents a serialized RayServeSyncHandle
+SERVE_SYNC_HANDLE_JSON_KEY = "__SerializedServeSyncHandle__"
```

Comment:
If the JSON is a dictionary anyways, probably preferable to have a single key and have a `sync` flag that indicates if it's sync or async